scripts/release_notes.rb

```diff
@@ -19,47 +19,77 @@
 @project_name = ARGV[0]
 @current_release_name = ARGV[1]
 
+# Additional projects to include in release notes
+# Both hammer-cli and hammer-cli-foreman versions are stored in the 'hammer-cli' project
+ADDITIONAL_PROJECTS = [
+  { project: 'hammer-cli', version_prefix: 'hammer-cli-' },
+  { project: 'hammer-cli', version_prefix: 'hammer-cli-foreman-' }
+].freeze
+
+# Category mappings and transformations
+CATEGORY_MAPPINGS = {
+  # Merge categories - source categories are merged into target category
+  merge: {
+    'Smart Proxy' => ['Smart Proxy - Core'],
+    'SELinux' => ['SELinux - General Foreman']
+  },
+
+  # Rename categories - simple string replacements using regex patterns
+  rename: {
+    /^Hammer CLI$/ => 'Hammer CLI',  # Keep as-is for base hammer-cli
+  },
+
+  # Project-specific category transformations
+  project_specific: {
+    'hammer-cli-foreman-' => {
+      /^Hammer CLI/ => 'Hammer CLI - Foreman'
+    }
+  }
+}.freeze
+
 def url_to_json(url)
   uri = URI(url)
   response = Net::HTTP.get(uri)
   JSON.parse(response)
 end
 
 # Determine version
-def get_version_id(project)
+def get_version_id(project, release_name = @current_release_name)
   url = "#{URL}/projects/#{project}/versions.json"
   result = url_to_json(url)
   return nil if result['versions'].nil?
-  result = result['versions'].detect{|v| v['name'] == @current_release_name }
+  # Find version with matching name (e.g., "3.16.0" or "hammer-cli-3.16.0")
+  result = result['versions'].detect{|v| v['name'] == release_name }
   return nil if result.nil?
   result['id']
 rescue StandardError => e
   puts "Error getting version list from #{url}: #{e}"
-  exit 1
+  return nil
```

Comment:
I think this should still exit if it fails. It may not be very clean, but it's a simple script and it's good enough error handling.
```suggestion
  exit 1
```